consensus/hotstuff/eventhandler/event_handler.go

```diff
@@ -279,11 +414,8 @@ func (e *EventHandler) startNewView() error {
 		if err != nil {
 			log.Warn().Err(err).Msg("could not forward proposal")
 		}
-		// mark our own proposals to avoid double validation
-		e.ownProposal = proposal.Block.BlockID
 
 		// We return here to correspond to the HotStuff state machine.
```

Comment:
https://github.com/onflow/flow-go/blob/542c0928f9f7f44264c4527688ec005c1568c495/consensus/hotstuff/eventhandler/event_handler.go#L418-L425

I feel this entire comment block is now outdated. Would suggest to just remove